receiver/windowsperfcountersreceiver/windowsperfcounters_scraper_test.go

```diff
@@ -188,37 +158,14 @@ func Test_WindowsPerfCounterScraper(t *testing.T) {
 			}
 			require.NoError(t, err)
 
-			if test.mockCounterPath != "" || test.scrapeErr != nil || test.shutdownErr != nil {
-				scraper.cfg.MetricMetaData = map[string]MetricConfig{
-					"metric": {
-						Description: "desc",
-						Unit:        "1",
-						Gauge:       GaugeMetric{},
-					},
-				}
-				scraper.counters = []PerfCounterMetrics{
-					{
-						CounterScraper: newMockPerfCounter(test.mockCounterPath, test.scrapeErr, test.shutdownErr),
-						Metric:         "metric",
-					},
-				}
-			}
-
 			actualMetrics, err := scraper.scrape(context.Background())
-			if test.scrapeErr != nil {
-				require.Equal(t, test.scrapeErr, err)
-				return
-			}
 			require.NoError(t, err)
 
 			err = scraper.shutdown(context.Background())
-			if test.shutdownErr != nil {
-				assert.Equal(t, test.shutdownErr, err)
-				return
-			}
+
 			require.NoError(t, err)
 			expectedMetrics, err := golden.ReadMetrics(test.expectedMetricPath)
-			scrapertest.CompareMetrics(expectedMetrics, actualMetrics, scrapertest.IgnoreMetricValues)
+			scrapertest.CompareMetrics(expectedMetrics, actualMetrics)
```

Comment:
Are we no longer retrieving actual data points here?

Reply:
I was getting aweird error with it being there saying it was an incompatible function and removed it but it seems to be working now.